osu.Game/Screens/Menu/IntroScreen.cs

```diff
@@ -201,6 +206,8 @@ public override void OnResuming(ScreenTransitionEvent e)
         {
             this.FadeIn(300);
 
+            ApplyToBackground(b => b.FadeColour(Color4.Black));
```

Comment:
Is it intentional that this is fading to black immediately? Previously `IntroTriangles` and `IntroWelcome` would perform it smoothly with 100ms duration.
```suggestion
            ApplyToBackground(b => b.FadeColour(Color4.Black, 100));
```